openapi/publicapi_spec.json

```diff
@@ -1829,6 +1844,16 @@
             "$ref" : "#/components/schemas/Link"
           }
         }
+      },
+      "ThreeDSecure" : {
+        "type" : "object",
+        "description" : "A structure representing 3ds summary",
+        "properties" : {
+          "required" : {
+            "type" : "boolean",
+            "description" : "Boolean indicating whether the payment was authenticated using 3DS."
```

Comment:
I would go for:

> Boolean indicating whether the payment required 3D Secure authentication.

I think this is more accurate because it is possible that the user started the payment but didn't get far enough in the process for the payment to be authenticated with 3DS. I think this is what @kbottla said in a chat I had with him last week!